tests/scripts/link_theater_and_siret_in_allocine_pivot_test.py

```diff
@@ -0,0 +1,20 @@
+from models import AllocinePivot
+from repository import repository
+from scripts.link_theater_and_siret_in_allocine_pivot import link_theater_to_siret
+from tests.conftest import clean_database
+from tests.model_creators.generic_creators import create_venue, create_offerer
+
+
+@clean_database
+def test_can_create_link_between_siret_and_theater(app):
+    # Given
+    offerer = create_offerer(siren='123456789')
```

Comment:
```suggestion
    offerer = create_offerer()
```

Le siren est mis par défaut dans le créateur